azure/terraform/README.md

```diff
@@ -31,10 +31,22 @@ This is used to help generate unique resource names for:
 * Azure storage account name
 Note: these resources must be globally unique across all of Azure.
 
+The template also provides the `deploy_airflow` and `deploy_argo` flags. These are booleans that specify if airflow or argo will be deployed in the kubernetes cluster along with metaflow related services. 
+
 Next, apply the `infra` module (creates Azure cloud resources only).
 
     terraform apply -target="module.infra"
 
+### Airflow Deployment with Azure
```

Comment:
We should communicate that Airflow will be used for orchestration somehow.  May be the header can mention it, or a one liner within the block.  Also we don't want to give the impression that Airflow is the primary and recommended goto orchestrator (that is Argo).  So perhaps something like this:

E.g. 
Orchestrating Metaflow jobs on Kubernetes
-------------------------------------------

The recommended way to orchestrate Metaflow workloads on Kubernetes is via Argo Workflows (LINK TO metaflow docs).  However, Airflow is also supported as an alternative (link to metaflow docs, once available).

Subtitle: Argo
Argo Workflows is installed by default on the AKS cluster as part of the "services" submodule.

Subtitle: Airflow
Blah blah blah

Reply:
Agreed. Will all the related docs.